examples/getting-started/README.md

```diff
@@ -40,87 +40,108 @@ The behaviour of the order processing back-end, in a nutshell, is shown below.
+Assume you have followed steps in the section, [Create the order processing back-end.](#1-create-the-order-processing-back-end)
    
 1. Copy the `order-process-be/order-processCompositeApplication/target/order-processCompositeApplication_1.0.0.car` into `<MI1_HOME>/repository/deployment/server/carbonapps`
 
 2. Start a Micro Integrator instance(MI1) with a [port offset.](../../doc/configuring-port-offset.md) (The reason why we start the server with a port offset is because we will be starting another instance of WSO2MI for the composite service to as explained in the [section below,](#deploying-the-composite-service-locally) leading to a port conflict.)
 
-3. You should be able to access the back-end using the below URL
+   **Note -** If you want to start Micro-Integrator with a portOffset=1 you have to pass the parameter as ```-DportOffset=11```, because Micro-Integrator starts with a default portOffset=10.
+      
+3. You should be able to access the back-end using the below curl command
    ```
-   http://localhost:8291/order
+   curl --header "Content-Type: application/json" --request POST  --data '{"store": {"book": [{"author": "Nigel Rees","title": "Sayings of the Century"},{"author": "J. R. R. Tolkien","title": "The Lord of the Rings","isbn": "0-395-19395-8"}]}}' http://localhost:8291/order
    ```
-##### Deploying the composite service locally
+#### Deploying the composite service locally
 
-Assume you have followed steps 1-4 in the section [Create the composite service to invoke the back-end.](#create-the-composite-service-to-invoke-the-back-end) The C-App containing the composite service resides in `composite-service/composite-service-capp/target`.
+Assume you have followed steps 1-4 in the section [Create the composite service to invoke the back-end.](#2-create-the-composite-service-to-invoke-the-back-end)
 
 1. Copy the `composite-service/composite-service-capp/target/composite-serviceCompositeApplication_1.0.0.car` into `<MI2_HOME>/repository/deployment/server/carbonapps`
 
 2. Start the Micro Integrator instance(MI2).
 
+You have successfully deployed 'order-process-backend' and the 'composite-service' services locally. Now you can test your deployments by using [Testing](#Testing) section.
+
+### Docker Deployment
+Please note that the distribution must be built with ```-Ddocker.skip=false``` prior to this step, for the base image to be created. And also assume that you have followed all the steps in the section [Create the order processing back-end.](#1-create-the-order-processing-back-end) and [Create the composite service to invoke the back-end.](#2-create-the-composite-service-to-invoke-the-back-end)
+In [Create the composite service to invoke the back-end](#2-create-the-composite-service-to-invoke-the-back-end) step 3 you need to replace 'order-process-backend' endpoint address as follows because in docker containers each container has their own localhost.
```

Comment:
Shall we use a link to point to the file https://github.com/wso2/micro-integrator/blob/master/examples/getting-started/composite-service/composite-service/src/main/synapse-config/api/forwardOrderApi.xml?

Reply:
Done